projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -91,6 +91,10 @@ export interface IComboSelectionChangeEventArgs extends CancelableEventArgs, IBa
     added: any[];
     /** An array containing the values that will be removed from the selection (if any) */
     removed: any[];
+    /** The text that will be displayed in the combo text box
+     * If the combo is **remote**, this will be empty
+     */
+    textValue: string;
```

Comment:
Something about `textValue` doesn't sit quite right with me. Maybe just `text` (the value bit is redundant) or `[input|display]Text` as a more descriptive option.
```suggestion
    : string;
```

Reply:
Would be best if this is consistent with how other properties/methods refer to the text in the main input.